### ISSUE TITLE
Cannot find name 'FactorId_2'

### ISSUE THREAD

**Issue by jeremyandes** [angular/angularfire#3329]
### Version info
**Angular: `^14.2.12`**
**AngularFire: `7.4.0 || 7.5.0`**
**Firebase: _any version_**

### How to reproduce these conditions
Just doing `npm i`, even when I do `rm -rf package-lock.json && rm -rf node_modules && npm i`

### Debug output
```
Error: node_modules/@firebase/auth/dist/auth-public.d.ts:3225:23 - error TS2304: Cannot find name 'FactorId_2'.
3225     static FACTOR_ID: FactorId_2;
```

**Screenshot**
<img width="1235" alt="image" src="https://user-images.githubusercontent.com/73316325/228965303-1038efd6-4b80-408a-bbff-445983633bf8.png">

### Expected behavior
To compile the app and run it on browser.

### Actual behavior
It doesn't compile.

**Comment by ricardoHer** [comment]
Facing the same issue here:

Angular: "^15.2.0"
AngularFire: "^7.5.0"

![image](https://user-images.githubusercontent.com/20401129/228973783-9309d6f6-4612-4971-a939-8f1137e328b3.png)

**Comment by jeremyandes** [comment]
This seems to be something about `TotpMultiFactorGenerator` class types in `auth-public.d.ts`.
Please check this link: https://github.com/firebase/firebase-js-sdk/issues/7174#issuecomment-1491919491

**Comment by jeremyandes** [comment]
@ricardoHer glad it worked for you!

**Comment by DimaIgorevich** [comment]
But it require to change package dependency. Because 7.5.0 install only 9.19.0 but hotfix already available in 9.19.1.

**Comment by jeremyandes** [comment]
Fixed on https://github.com/firebase/firebase-js-sdk/issues/7174#issuecomment-1492607813
